app/schemas/post.py

```
@@ -0,0 +1,172 @@
+                    "title": "수정된 제목",
+                    "content": "수정된 내용입니다..."
+                }
+            ]
+        }
+    }
+
+class PostAuthorInfo(UserAuthorInfo):
+    """
+    게시글 작성자 정보 (UserAuthorInfo 상속)
+    """
+    pass
```

Ruff (0.14.11)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)